src/w3c/headers.js

```diff
@@ -501,17 +501,16 @@ export function run(conf, doc, cb) {
       $("<meta lang='' property='dc:language' content='en'>")
     );
   }
-  // insert into document and mark with microformat
-  var bp;
-  if (conf.isCGBG) bp = cgbgHeadersTmpl(conf);
-  else bp = headersTmpl(conf);
-  $("body", doc)
-    .prepend($(bp))
-    .addClass("h-entry");
+  // insert into document
+  document.body.insertBefore(
+    (conf.isCGBG ? cgbgHeadersTmpl : headersTmpl)(conf),
+    document.body.firstChild
+  );
+  document.body.classList.add("h-entry");
```

Comment:
Are you sure this is correct? In the previous code, I looks like it was adding the "h-entry" to the "bp", instead of the body.

Reply:
fixed typo, "it looks like"

Reply:
At first I thought so, but try this:

```js
$("<div>").prepend($("<a>"))[0];
// returns <div>
```

`.prepend()` returns the container instead of the child.